src/librustc/dep_graph/dep_node.rs

```diff
@@ -148,6 +148,10 @@ pub enum DepNode<D: Clone + Debug> {
     // For proj. cache, we just keep a list of all def-ids, since it is
     // not a hotspot.
     ProjectionCache { def_ids: Vec<D> },
+
+    // Depnodes for MetaData
```

Comment:
Ah that makes sense the title of the issue was deceiving. I assumed all of these were just about crate metadata.